Tests/EventListener/ExceptionLogSubscriberTest.php

```diff
@@ -98,4 +99,15 @@ public function testOnExceptionImproperEvent(): void
 
         $this->subject->onException(new stdClass());
     }
+
+    public function testProvidingDefaultExceptionMessageAsLogsCanNotBeEmpty(): void
+    {
+        $this->tracing->logInActiveSpan(Argument::that(function (array $argument) {
+            self::assertArrayHasKey('message', $argument);
+            self::assertEquals(ExceptionLogSubscriber::DEFAULT_ERROR_MESSAGE, $argument['message']);
```

Comment:
`ExceptionLogSubscriber::DEFAULT_ERROR_MESSAGE` won't be available here, if we change the `const` to `private const` there.

```suggestion
            self::assertEquals("No error message given", $argument['message']);
```